src/test/ui/binding/issue-53114-safety-checks.rs

```diff
@@ -22,7 +22,7 @@ fn let_wild_gets_unsafe_field() {
     let p = P { a: &2, b: &3 };
     let _ = &p.b;  //~ ERROR    reference to packed field
     //~^  WARN will become a hard error
-    let _ = u1.a;  // #53114: should eventually signal error as well
+    let _ = u1.a;  //~ ERROR  [E0133]
```

Comment:
@est31 this is the test for #54003.

Reply:
I meant one with `_ =` only, without a `let`.